hardware/products/ucm/g400.md

```diff
@@ -1,5 +1,250 @@
 # G400
 ![G400S](images/g400s.jpg) ![G400D](images/g400d.jpg)
+# Overview
+---
+The G400-D System on Module (SoM) is the precursor to the Universal Compute Modules (UCM). It pioneered the SO-DIMM System on Module (SoM) originally running .Net Micro Framework exclusively. It now can operate various other platforms, including TinyCLR.
+
+# Specifications
+---
+|                |                                              |
+| :------------- | :------------------------------------------- |
+| Processor      | AT91SAM9X35 (Atmel/Microchip 32-bit ARM 926) |
+| Speed          | 400 MHz                                      |
+| RAM            | 32 KB (SRAM)                                 |
+| Flash          | Not Supported                                |
+| External RAM   | 128 MB (DDR2 SDRAM)                          |
+| External Flash | 4 MB (SPI)                                   |
+| UART           | 1 UCM Standard (5 overall\* including HS)    |
+| UART HS        | 1 UCM Standard                               |
+| I2C            | 1 UCM Standard                               |
+| SPI            | 1 UCM Standard                               |
+| CAN            | 1 UCM Standard (2 overall\*)                 |
+| SDIO           | 1 UCM Standard                               |
+| ADC            | 8 UCM Standard                               |
+| PWM            | 4 UCM Standard                               |
+| GPIO           | 12 UCM Standard (70 overall\*)               |
+| IRQ            | 4 UCM Standard (70 overall\*)                |
+| USB Client     | Supported                                    |
+| USB Host       | Supported (2 overall\*)                      |
+| LCD            | Supported (16bpp)                            |
+| Ethernet PHY   | Supported                                    |
+| WiFi           | Not supported                                |
+| DCMI           | Not supported                                |
+| VBAT           | Supported                                    |
+| JTAG           | Full JTAG                                    |
+*\*overall means that there are more of that feature multiplexed with other pins/features*
```

Comment:
I think it's a lot clearer to have a third column for the total supported and keep the second column just for UCM count, especially so we don't repeat UCM Standard a lot. We can have two tables where the first is just the specs that don't change then a second for peripheral counts.

Reply:
I would agree that another column would be better. However, because we cannot format these tables properly with col span and the such, it is harder to deal with the tables in such a small space. I believe the table columns will not be big enough for the larger labels.

Reply:
This will be more of an issue with the UC5550 as it already has three columns. We will need five to separate this way.